src/components/LatestMovies/LatestMovies.module.css

```
@@ -0,0 +1,147 @@
+  to {
+    transform: translateX(0);
+    opacity: 1;
+  }
+}
+
+.movie {
+  flex: 0 0 auto;
+  text-align: center;
+  position: relative;
+  padding-left: 47px;
+  flex: 0 0 auto;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

flex is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)